### PR TITLE
[WFLY-8115] added directory and testcases for jaxrs

### PR DESCRIPTION
**Pull request by rsearls**

### CODE REVIEW

File: testsuite/integration/jaxrs/pom.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        <jbossas.ts.integ.dir>${basedir}/..</jbossas.ts.integ.dir>
+    </properties>
+
+    <profiles>
+
+        <profile>
+            <id>jaxrs.integration.tests.profile</id>
+            <activation>
+                <property>
+                    <name>!no.jaxrs.integration.tests</name>
```
Comment (bstansberry) [inline testsuite/integration/jaxrs/pom.xml]: Should the activation of this be the same as the other non-smoke suites in testsuite/integration; i.e. turned on via -Dts.jaxrs (so off by default)?

**Comment by marekkopecky** [comment]
@rsearls (cc: @asoldano, @ctomc ): Is it necessary to add new maven module to TS? Are there some specific pom.xml configuration in this new jax-rs module? Can these tests be located with other jax-rs tests in basic module? See https://github.com/wildfly/wildfly/tree/master/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jaxrs

File: testsuite/integration/jaxrs/pom.xml (hunk 1 above)
Comment (rsearls) [inline testsuite/integration/jaxrs/pom.xml]: This is more closely related to ws than clustering or iiop.  I was following ws as the model.  However a -Dts.jaxrs could be added.

**Comment by rsearls** [comment]
These tests require the use of arquillian.  I do not see a way to use that in the base module.  Is there a way?

**Comment by marekkopecky** [comment]
Basic module uses arquillian too, [see this example](https://github.com/wildfly/wildfly/blob/master/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jaxrs/jackson/JaxrsJacksonProviderTestCase.java)

**Comment by rsearls** [comment]
OK I can move this tests into testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jaxrs
Is that what is preferred?

**Comment by rsearls** [comment]
Moving tests into testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jaxrs

**Pull request closed without merging**